src/test/java/org/tests/query/other/TestQuerySingleAttribute.java

```diff
@@ -100,6 +106,65 @@ public void findSingleAttribute() {
     assertThat(sqlOf(query)).contains("select t0.name from o_customer t0");
     assertThat(name).isNotNull();
   }
+  
+  @Test
+  public void findSingleAttributeList_with_join_column() {
+
+    ResetBasicData.reset();
+
+    MainEntity e1 = new MainEntity();
+    e1.setId("1");
+    e1.setAttr1("a1");
+    DB.save(e1);
+    
+    MainEntity e2 = new MainEntity();
+    e2.setId("2");
+    e2.setAttr1("a2");
+    DB.save(e2);
+    
+    MainEntity e3 = new MainEntity();
+    e3.setId("3");
+    e3.setAttr1("a3");
+    DB.save(e3);
+    
+    MainEntityRelation rel = new MainEntityRelation();
+    rel.setEntity1(e1);
+    rel.setEntity2(e2);
+    DB.save(rel);
+    
+    rel = new MainEntityRelation();
+    rel.setEntity1(e1);
+    rel.setEntity2(e3);
+    DB.save(rel);
+    
+    rel = new MainEntityRelation();
+    rel.setEntity1(e2);
+    rel.setEntity2(e3);
+    DB.save(rel);
+    
+    Query<MainEntityRelation> query = Ebean.find(MainEntityRelation.class)
+      .fetch("entity1", "attr1")
+      .setDistinct(true)
+      .setCountDistinct(CountDistinctOrder.COUNT_DESC_ATTR_ASC)
+      .where().query();
+
+    List<CountedValue<String>> attr1list = query.findSingleAttributeList();
+
+    assertThat(sqlOf(query)).contains("select distinct r1.attribute_1, r1.attribute_2, count(*) cnt from "
+        + "(select t0.id1 attribute_1, t1.attr1 attribute_2 from main_entity_relation t0 "
+        + "left join main_entity t1 on t1.id = t0.id1 ) r1 "
+        + "group by r1.attribute_1, r1.attribute_2 "
+        + "order by count(*) desc, r1.attribute_1");
```

Comment:
I think, this query is wrong. There should be only ONE attribute `r1.attribute_2`=`t1.attr1` but there is a 2nd: `r1.attribute_1`=`t0.id` which would even produce a wrong result. (every attr1 of every t0 root bean would be counted)

@AlexWagner Maybe we must explicitly deselect the root bean with `.select("").fetch("entity1","attr1")`
This can also be a regression due the changes for @ThomasFOC dashboard queries.
Can you check this with 11.41.x or against official ebean release?